# This message describes the pose error
# There is one for each x , y and theta(orientaion) for describing the error
float32 x_error
float32 y_error
float32 theta_error
